code/modules/heavy_vehicle/mecha.dm

```diff
@@ -165,6 +175,28 @@
 	if(head && head.radio)
 		radio = new(src)
 
+	// Remote Controlled shenanigans
```

Comment:
Move that to a different proc, so its possible to reuse it in case we want to add the option to build remote controlled mechs / use it for adminbus